frontend/packages/ceph-storage-plugin/integration-tests-cypress/consts.ts

```diff
@@ -0,0 +1,110 @@
+export const OCS_OP = 'OpenShift Container Storage';
```

Comment:
If these tests are specific to OCS, they should go into the plugin package directory. @dtaylor113 can give you more details on how this works.

Reply:
Disregard, I missed that these tests were already in the package.